libraries/chain/include/eosio/chain/hotstuff.hpp

```diff
@@ -72,7 +97,7 @@ namespace eosio::chain {
       fc::sha256 b_finality_violation;
       block_id_type block_exec;
       block_id_type pending_proposal_block;
-      uint32_t v_height = 0;
+      eosio::chain::view_number v_height;
```

Comment:
I think this and other members should be `std::optional`, but we can update that later.